SPECS/telegraf/telegraf.spec

```
@@ -33,8 +34,10 @@ the community can easily add support for collecting metrics from well known serv
 Postgres, or Redis) and third party APIs (like Mailchimp, AWS CloudWatch, or Google Analytics).
 
 %prep
-%autosetup -p1
+%autosetup -N
+# Apply vendor before patching
 tar -xf %{SOURCE1}
+%autopatch -p1
```

Out of curiosity, can you try if this will work to simplify cases with vendor tarballs?
```suggestion
%autosetup -a 1 -p1
```

---

And it's not a blocker, just a friendly request, if you have time.